roles/container-storage-setup/templates/docker-storage-setup-overlayfs.j2

```diff
@@ -0,0 +1,11 @@
+{% if docker_storage_mountpoints is defined %}
+DEVS="{{ docker_storage_mountpoints }}"
+{% else %}
+DEVS="{{ openshift_openstack_container_storage_setup.docker_dev }}"
```

Comment:
same as above for these - make sure they don't have `openstack` as part of the variable names